versioned_docs/version-8.7/components/modeler/web-modeler/process-application-pipeline.md

```diff
@@ -91,11 +93,20 @@ After the review is complete, you can promote the versioned process application
 If you want to use your own deployment pipeline after the review is complete, you can use [Git Sync](git-sync.md) at this point to deploy and promote the process application through your own pipeline.
 :::
 
+### Deployment policy
+
+By default, only organization administrators can deploy process applications to the `prod` tagged clusters.
+Organization administrators can change this policy in the [process application deployment settings](/docs/components/modeler/web-modeler/web-modeler-settings.md#process-application-deployment).
+There, you can enable non-admin users with deployment permission to deploy to process applications to production stages
+after a collaborator has the approved process application version using the [process application version review](process-application-pipeline.md#review).
```

Comment:
> you can enable non-admin users with deployment permission to deploy to process applications to production stages after a collaborator has the approved process application version using the [process application version review]

So, non-admin users with deployment permissions can deploy process applications to prod after a collaborator has approved the process app version?

Reply:
Yes. Correct;